### DISCUSSION TITLE
Theme for Mac

### DISCUSSION THREAD

**Discussion by MrWelsch** [Cutwell/VS-catppuccin-theme#6] [Ideas]
Hello,

I really like this theme as I am using it on pretty much every tool.

Now I am required to use Visual Studio and since I am on a mac I cannot install it via vsix.

Hence the question if it would be possible to release a mac version of this great theme?

**Comment by Cutwell** [comment]
Good question - the Mac OSX and Windows OS versions of Visual Studio 2022 are completely different apps, meaning .vsix files do not work.
I don't use VS 2022 on Mac OSX, do they support themes at all at presents? If so it may be possible to create the theme from scratch to target the app, otherwise best option may be to submit a ticket and wait for development to provide cross-compatability.
